doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -237,44 +237,54 @@ analysis on inductive or co-inductive objects (see :ref:`variants`).
-   inductive type and the tactic :n:`induction` generates subgoals, one for
-   each possible form of :n:`@term`, i.e. one for each constructor of the
-   inductive type.
+   Applies an :term:`induction principle` to generate a subgoal for
+   each constructor of an inductive type.
 
-   If the argument is dependent in either the conclusion or some
+   If the argument is :term:`dependent <dependent product>` in the conclusion or some
    hypotheses of the goal, the argument is replaced by the appropriate
-   constructor form in each of the resulting subgoals and induction
+   constructor in each of the resulting subgoals and induction
    hypotheses are added to the local context using names whose prefix
-   is **IH**.
-
-   There are particular cases:
-
-   + If term is an identifier :n:`@ident` denoting a quantified variable of the
-     conclusion of the goal, then inductionident behaves as :n:`intros until
-     @ident; induction @ident`. If :n:`@ident` is not anymore dependent in the
-     goal after application of :n:`induction`, it is erased (to avoid erasure,
-     use parentheses, as in :n:`induction (@ident)`).
-   + If :n:`@term` is a :n:`@natural`, then :n:`induction @natural` behaves as
+   is **IH**.  The tactic is similar to :tacn:`destruct`, except that
+   `destruct` doesn't generate induction hypotheses.
+
+   There are several cases:
+
+   + If :n:`@one_term` (in :n:`@induction_arg`)
+     is an identifier :n:`@ident` denoting a quantified variable of the
+     conclusion of the goal, then :n:`induction @ident` behaves like :tacn:`intros` :n:`until
+     @ident; induction @ident`. If :n:`@ident` is no longer dependent in the
+     goal after application of :n:`induction`, it is erased. To avoid erasure,
+     use parentheses, as in :n:`induction (@ident)`.
+   + If :n:`@induction_arg` is a :n:`@natural`, then :n:`induction @natural` behaves like
      :n:`intros until @natural` followed by :n:`induction` applied to the last
      introduced hypothesis.
 
      .. note::
-        For simple induction on a number, use syntax induction (number)
-        (not very interesting anyway).
+        For simple induction on a number, use the form `induction (number)`
+        (which is not very interesting).
 
-   + In case term is a hypothesis :n:`@ident` of the context, and :n:`@ident`
-     is not anymore dependent in the goal after application of :n:`induction`,
+   + If :n:`@one_term` is a hypothesis :n:`@ident` of the context, and :n:`@ident`
+     is no longer dependent in the goal after application of :n:`induction`,
      it is erased (to avoid erasure, use parentheses, as in
      :n:`induction (@ident)`).
-   + The argument :n:`@term` can also be a pattern of which holes are denoted
+   + The argument :n:`@one_term` can also be a pattern in which holes are denoted
      by “_”. In this case, the tactic checks that all subterms matching the
      pattern in the conclusion and the hypotheses are compatible and
```

Comment:
To be changed like in the case of `destruct`. It doesn't do this check. It just finds the first pattern that matches. As in this example:

```coq
Goal forall n m, n + m = m + n.
intros.
induction _.
```